docs/suppressing-instrumentation.md

```diff
@@ -72,9 +75,11 @@ corresponding instrumentation `name`:
 | OkHttp | okhttp|
 | OpenLiberty | liberty |
 | OpenTelemetry Trace annotations | opentelemetry-annotations |
+| OpenTelemetry SDK | opentelemetry-api |
```

Comment:
since we are instrumenting specific portions of the OpenTelemetry SDK (e.g. also `opentelemetry-annotations` above)
```suggestion
| OpenTelemetry API | opentelemetry-api |
```